tests/oss-fuzz/repro_fuzz.cc

```diff
@@ -1,12 +1,16 @@
 // Copyright 2023 Google LLC
 // SPDX-License-Identifier: BSD-2-Clause
 
+#include <chrono>
 #include <cstddef>
 #include <cstdint>
 #include <fstream>
 #include <iostream>
 #include <vector>
 
+using seconds = std::chrono::duration<double>;
+using chrono = std::chrono::high_resolution_clock;
```

Comment:
"clock" would be a clearer type alias. "chrono" looks like "std::chrono".

Also these two type aliases probably should follow the naming convention of the Google C++ Style Guide?